.env.testing.example

```
@@ -0,0 +1,83 @@
+APP_NAME=Heartdevs
+APP_ENV=testing
+APP_KEY=base64:vCE9NeWv1bkp19JvhTCeMk+qPEC52Vim7shJUljveCg=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_KEY key should go before the APP_NAME key

(UnorderedKey)